vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,53 @@
+# Entando Glossary
+
+## Entando Terms
+
+| Term |  Description
+|:--|:--
+| Entando Application | An application built with the Entando Platform and available at a given URL |
+| Entando App Builder | The frontend of the Entando Platform that hosts the Entando WCMS and provides a feature-rich user interface to configure and interact with components, design and create pages, manage content, and build applications |
+| Entando App Engine | The core runtime engine responsible for the primary out-of-the-box services required to develop Entando Applications; it exposes the Entando App Builder APIs, assembles and coordinates components within the Entando App Builder, manages the cluster infrastructure, and provides the data access layer to persist the page and application design |
+| Entando Bundle | A packaged set of components and resources created on Entando |
+| Entando CLI (ent CLI) | Entando's core command line interface |
+| Entando Cloud Hub | A SaaS instance of an Entando Hub that contains a public and private collection of components |
+| Entando Cluster | The infrastructure Entando deploys on Kubernetes via the Entando Operator and controllers; each element is associated with a custom resource definition, if applicable |
+| Entando Component Generator (ECG) | Entando's implementation of a JHipster blueprint that generates the components used to build an Entando Application via automation and templating |
+| Entando Component Manager (ECM) | Provides functionality to build and organize micro frontends and widgets using the App Builder and manages the connections between an application and the installed microservices |
+| Entando Content | A structured element representing a set of information built using Entando Content Attributes and added to an Entando Application via the Entando App Builder |
+| Entando Content Type | The structure of specific content, defined by a set of content attributes |
+| Entando Content Attribute | The basic data that define a content type |
+| Entando Content Template | The style or layout of a content type where a single content type can have multiple content models defining different ways to render the same content |
+| Entando Digital Asset | An image, document, or other media file that is uniquely identifiable and stored digitally in a format supported by the Entando Platform |
+| Entando Hub |  A repository (local, remote, public, or private) containing components built with the Entando Platform; A single Entando App Builder can connect to 1 or more Entando Hubs |
+| Entando Identity Management | Entando's Keycloak- and token-based user management and authentication system |
```

Comment:
I would simplify to "Keycloak-based". Token implies OAuth but it's odd to include it here.